openlibrary/coverstore/coverlib.py

```diff
@@ -55,6 +55,7 @@ def make_path_prefix(olid, date=None):
     return "%04d/%02d/%02d/%s-%s" % (date.year, date.month, date.day, olid, random_string(5))
 
 def write_image(data, prefix):
+    # type: (bytes, str) -> Optional[Image]
```

Comment:
This is throwing a lint error (!)
```suggestion
    # type: (bytes, str) -> Image
```